src/vstart.sh

```diff
@@ -1224,13 +1224,23 @@ if [ "$new" -eq 1 ]; then
     echo ""
 fi
 echo ""
-echo "export PYTHONPATH=./pybind:$PYTHONPATH"
-echo "export LD_LIBRARY_PATH=$CEPH_LIB"
+# add header to the environment file
+{
+    echo "#" > $CEPH_DIR/vstart_environment.sh
```

Comment:
You can remove the ` > $CEPH_DIR/vstart_environment.sh` from all of these echo commands now :) That was the point of the suggested change.

Reply:
\<facepalm\>